packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/aggregations/aggs_types/bucket_aggs.ts

```diff
@@ -61,6 +61,36 @@ const existsSchema = s.object({
     })
   ),
 });
+
+const rangeSchema = s.object({
+  range: s.recordOf(
+    s.string(),
+    s.object({
+      lt: s.maybe(s.string()),
+      lte: s.maybe(s.string()),
+      gt: s.maybe(s.string()),
+      gte: s.maybe(s.string()),
+    })
+  ),
+});
+
+const termValueSchema = s.object({
+  term: s.recordOf(s.string(), s.object({ value: s.string() })),
+});
+
+const nestedSchema = s.object({
```

Comment:
These changes are needed for the filter here: https://github.com/elastic/kibana/pull/147295/files#diff-94dd88fd0956fc92263b02e7ea005e640da97008a8c22d7acd8a13fe2eb11c70R168